src/jwks.cc

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include "jwt_verify_lib/jwks.h"
```

Comment:
duplicated with line 24

Reply:
Done, leaving include where it is for consistency with other files in this repo

Reply:
Actually check-style is forcing this, which is consistent with Google C++ style, so leaving as is.